pkgs/by-name/au/authentik/package.nix

```diff
@@ -161,16 +161,6 @@ let
     packageOverrides = final: prev: {
       django = final.django_5;
 
-      django-tenants = prev.django-tenants.overrideAttrs {
-        version = "3.7.0-unstable-2025-03-14";
-        src = fetchFromGitHub {
-          owner = "rissson";
-          repo = "django-tenants";
-          rev = "156e53a6f5902d74b73dd9d0192fffaa2587a740";
-          hash = "sha256-xmhfPgCmcFr5axVV65fCq/AcV8ApRVvFVEpq3cQSVqo=";
-        };
-      };
```

Comment:
I'm pretty sure that this will break authentik too, since afaict django-tenants is still on 3.7, but we'd need at least 3.8 for authentik.